### ISSUE TITLE
Daily - Viernes 24/10/2025

### ISSUE THREAD

**Issue by nikaLPFB** [BreakfastProject/proyecto7_ensemble_grupo2#22]
# Dailies Standup - Viernes 24/10/2025

## Asistentes
@MariaDunaeva1  @LadyFB89  @Yedpt  @CristianYepes 

### @MariaDunaeva1 
- [ ] Implementacion de CNN y del Modelo LightGBM
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
      matrices de confusión, etc.)
- [ ] Optimización del modelo con ajuste de hiperparámetros (GridSearch, RandomSearch, Bayesian
      Optimization, Optuna).

### @Yedpt 
- [ ] Implementacion de  CNN y del  Modelo XGBoost 
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
   matrices de confusión, etc.)
- [ ] Optimización del modelo con ajuste de hiperparámetros (GridSearch, RandomSearch, Bayesian
      Optimization, Optuna).

### @nikaLPFB 
- [ ] Implementacion de CNN  y luego Random Forest
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
   matrices de confusión, etc.)
- [ ] Optimización del modelo con ajuste de hiperparámetros (GridSearch, RandomSearch, Bayesian
      Optimization, Optuna).
- [ ] Realizar el Dailies diario

###  @CristianYepes 
- [ ]  Implementacion de CNN y Fast API
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
   matrices de confusión, etc.)

## Observaciones
Cristian , se conecta por las tardes razones  de estudio

Viernes 24/10/2025
Asignado a: Lady Flores Berrocal
Etiquetas: daily, standup
Estado: Abierto
